travisbuild.sh

```diff
@@ -8,12 +8,18 @@ export CUBERITE_BUILD_DATETIME=`date`
 
 # Use ccache if available
 if [ `which ccache` ]; then
+	# Re-run compile on pre-processed sources on cache miss (slightly faster?)
```

Comment:
It's slower actually, but clang builds fail without it.

Reply:
I'm starting to think all software sits on a huge creaking tower four miles high and two cm thick.

Reply:
Comment changed